### ISSUE TITLE
Error for automatic syntax finding for FORM file extension "*.frm"

### ISSUE THREAD

**Issue by mgerlach-phys** [vim/vim#9673]
### Steps to reproduce

1. Write .frm file
``` bash
echo "* do nothing" > test.frm
```
2. Try to open it with vim

### Expected behaviour

`FORM` syntax highlighting should be automatically found and loaded

### Version of Vim

8.2 (2019 Dec 12)

### Environment

OS: openSUSE Leap 15.3
Terminal: urxvt via ssh/tmux
Value of $TERM: screen
Shell: zsh

### Logs and stack traces

It prints the following when I try to open the file
```shell
Error detected while processing /usr/share/vim/vim82/suse.vimrc[10]../usr/share/vim/vim82/syntax/syntax.vim[42]..BufRead Autocommands for "*.frm":
E117: Unknown function: dist#ft#FTVB
```

### Hints
It is probably due to the refactoring of 6517f14165cdebf83a07ab9d4aeeb102b4e16e92 where [this line](https://github.com/vim/vim/blob/master/runtime/filetype.vim#L2054) was forgotten to change as well.

**Comment by lacygoill** [comment]
How about this patch?
```diff
diff --git a/runtime/autoload/dist/ft.vim b/runtime/autoload/dist/ft.vim
index f6ef9be89..37782a9fe 100644
--- a/runtime/autoload/dist/ft.vim
+++ b/runtime/autoload/dist/ft.vim
@@ -67,7 +67,7 @@ func dist#ft#FTasmsyntax()
   endif
 endfunc
 
-func dist#ft#FTbas()
+func dist#ft#FTbas(alt = '')
   if exists("g:filetype_bas")
     exe "setf " . g:filetype_bas
     return
@@ -88,6 +88,8 @@ func dist#ft#FTbas()
     setf qb64
   elseif match(lines, '\cVB_Name\|Begin VB\.\(Form\|MDIForm\|UserControl\)') > -1
     setf vb
+  elseif a:alt != ''
+    exe 'setf ' .. a:alt
   else
     setf basic
   endif
diff --git a/runtime/filetype.vim b/runtime/filetype.vim
index bc85b128a..01c200f98 100644
--- a/runtime/filetype.vim
+++ b/runtime/filetype.vim
@@ -2051,7 +2051,7 @@ au BufRead,BufNewFile *.hw,*.module,*.pkg
 	\ endif
 
 " Visual Basic (also uses *.bas) or FORM
-au BufNewFile,BufRead *.frm			call dist#ft#FTVB("form")
+au BufNewFile,BufRead *.frm			call dist#ft#FTbas('form')
 
 " SaxBasic is close to Visual Basic
 au BufNewFile,BufRead *.sba			setf vb
```

**Comment by brammool** [comment]
I'll ask Doug how to fix this.  The patch at least works for now.

**Comment by lacygoill** [comment]
Fixed in patch [8.2.4274](https://github.com/vim/vim/releases/tag/v8.2.4274)

**Comment by mgerlach-phys** [comment]
That was quick, thanks guys!
